compare.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+libnetworkPath="/home/flaviocrisciani/macos/src/github.com/docker/libnetwork"
```

Comment:
😇 Path doesn't exist on my machine; perhaps run this in a container? This could be a Make target, and have it fetch https://raw.githubusercontent.com/moby/moby/master/vendor.conf directly

Reply:
Also see my "inception bot" wish-list here: https://github.com/moby/moby/issues/35490